meson.build

```diff
@@ -544,6 +544,21 @@ if get_option('libarchive')
   plugin_deps += libarchive
 endif
 
+# plugin_logitech_bulkcontroller not supported on Windows
+# plugin_logitech_bulkcontroller depends on libprotobuf-c
+if get_option('plugin_logitech_bulkcontroller')
+    if host_machine.system() == 'windows'
+      error('plugin_logitech_bulkcontroller plugin not supported on Windows')
```

Comment:
does it not work on windows because of the lack of protobufc in mingw32/64 or some other reason?

Reply:
Some other reason, don't know full picture, but Licensing/Support etc...

Reply:
If it's licensing/support my preference would be some magic that for the tagged EXE releases we don't build it, but at least let CI build it.  We can probably detect this using the CI environment variable.
Then we'll know if we have technical problems with the cross compile.

Reply:
CI building Windows is fine